packages/preview-middleware/src/types/index.ts

```diff
@@ -27,7 +27,7 @@ export interface App {
  */
 export interface FlpConfig {
     path: string;
-    intent?: Intent;
+    intent: Intent;
```

Comment:
If this changes, then we need to update the README

Reply:
See above comment

Reply:
👍